### DISCUSSION TITLE
how do i run the aimbot

### DISCUSSION THREAD

**Discussion by TTVMyForce** [ttwizz/Open-Aimbot#2] [Q&A] [answered]
how do i run the aimbot

**Comment by JustMerely** [comment, marked as answer]
```lua
loadstring(game:HttpGet("https://raw.githubusercontent.com/ttwizz/Open-Aimbot/master/source.lua", true))()
```

**Comment by FLUFFYKATZA** [comment]
Hi, 

sorry i am new. How exactly do we open the app. I can't see an exe or anything.

**Reply by ttwizz** [reply]
It's Internal Aimbot and you have to execute the script.

**Reply by FLUFFYKATZA** [reply]
So can Ghub launch the script?

or i need a specific program. 

Can you please explain to me, an idiot, how to get it working?

i would like to try your program
